.github/pull_request_template.md

```diff
@@ -7,30 +7,24 @@ https://github.com/lucyparsons/OpenOversight/blob/develop/CONTRIB.md#recommended
 Also make sure you have read and abide by the code of conduct:
 https://github.com/lucyparsons/OpenOversight/blob/develop/CODE_OF_CONDUCT.md
 
+If this pull request is not ready for review yet, please submit it as a draft.
```

Comment:
It being a draft is a much better indicator of it being ready for review than a comment in the PR description.